### DISCUSSION TITLE
Nuxt 3 support?

### DISCUSSION THREAD

**Discussion by eMeRiKa13** [vue-final/vue-final-modal#187] [Q&A] [answered]
Hello,

One question, before I try myself, if someone has the answer.

Is-this component working with Nuxt 3 (released in beta last month)?

Thank you

**Comment by hunterliu1003** [comment]
I don’t know. I haven’t tried yet haha

**Comment by hunterliu1003** [comment, marked as answer]
@eMeRiKa13 
Here is the Demo that using vue-final-modal in Nuxt3 
https://codesandbox.io/s/blissful-spence-nnlw3
It just work!

**Reply by eMeRiKa13** [reply]
Awesome, thank you for the test (and your library!)

**Comment by boateng-ampaw** [comment]
I am new to Nuxt and tried integrating the vue final modal 4 but couldn't get it to work. I created the plugin file and added the css to the nuxt.config file, but now I am unable to add it to my pages or components as all the examples given seem to be in typescript which I am unfortunately familiar with.
I thought I could just import the ModalContainer add add my content like so:
```
<script setup>
import { ModalsContainer, useModal } from 'vue-final-modal'

const { open, close } = useModal()

</script>


<template>
<button @click="open">Show modal</button>

<ModalsContainer>
.... Any content goes here

<button @click="close"></button>
</ModalsContainer>

</template>
```

Any pointers on what to do ? Thanks

**Reply by hunterliu1003** [reply]
@boateng-ampaw Please read the basics usage of `useModal` in the [documentation](https://vue-final-modal.org/get-started/guide/setup)
